packages/mobile/src/components/AccountNumber.tsx

```diff
@@ -41,39 +41,42 @@ export default function AccountNumber({ address, touchDisabled, location }: Prop
   // Turns '0xce10ce10ce10ce10ce10ce10ce10ce10ce10ce10'
   // into 'ce10 ce10 ce10 ce10 ce10 ce10 ce10 ce10 ce10 ce10'
   const addressChunks = getAddressChunks(address)
+  const addressString = '0x ' + addressChunks.reduce((x, y) => x + ' ' + y)
```

Comment:
This didn't have any tests but it'd probably be good to add some :D Maybe just checking that the address is populated correctly with the right spaces and that on press the `Clipboard.setString` function gets called with the right parameter?